### DISCUSSION TITLE
抓走你们的麻麻！

### DISCUSSION THREAD

**Discussion by Zeteor** [share121/inter-knot#1075] [其他]
投降吧，我要将你们的麻麻都抓走
![Screenshot_2024-08-10-22-18-34-151_com opera browser-edit](https://github.com/user-attachments/assets/8b68a101-9a77-4834-812d-e21a8e9fe7d5)

**Comment by WORLD-executeME** [comment]
![672d8b22dd54564e68bf64f2f6de9c82d3584f9a](https://github.com/user-attachments/assets/45ec1b53-7b11-4c03-921a-15c67939a1e5)
